index.html

```diff
@@ -15,12 +15,19 @@
         <header>
             <div class="my-logo"><a href="index.html">Naan</a></div>
             <nav class="collapsed_menu">&#9776;</nav>
```

Comment:
- [ ] [OPTIONAL] Since this is a responsive website, when viewed with mobile, the menu icon is not well aligned to the right. Please consider giving the menu icon the correct margin to the right as shown below 👇🏾 

Your project
![image](https://user-images.githubusercontent.com/84156714/221579735-656e8f1a-9802-4ae2-b84c-4d047797d971.png)

Figma Design
![image](https://user-images.githubusercontent.com/84156714/221579852-781f3c01-1fca-4115-b395-dbd72955e77e.png)